src/components/Chart.jsx

```diff
@@ -66,49 +66,52 @@ class Chart extends Component {
         } = this.props;
 
         const currentPosition = `cq-chart-control-${(position && !isMobile) ? position : 'bottom'}`;
-        const currentMode = `${isMobile ? 'smartcharts-mobile' : ''}`;
         const contextWidth =  !isMobile ? `smartcharts-${containerWidth}` : '';
         const renderTopWidgets = topWidgets || defaultTopWidgets;
 
         return (
-            <cq-context
-                ref={(root) => { this.root = root; }}
-                class={`smartcharts-${theme} ${contextWidth}`}
+            <div
+                className={`smartcharts smartcharts-${theme} ${contextWidth} smartcharts-${isMobile ? 'mobile' : 'desktop'}`}
+                ref={(modalNode) => { this.modalNode = modalNode; }}
```

Comment:
the heart of the change is here. The portals will place the modal dropdowns in this div